npm/src/controller/oauth.ts

```diff
@@ -188,14 +189,18 @@ export class OAuthController implements IOAuthController {
       }
     } else if (
       (client_id && client_id !== '' && client_id !== 'undefined' && client_id !== 'null') ||
-      (access_type && access_type !== '' && access_type !== 'undefined' && access_type !== 'null')
+      (access_type && access_type !== '' && access_type !== 'undefined' && access_type !== 'null') ||
```

Comment:
This check for access_type and scope can probably be removed since we check for it before calling `getEncodedTenantProduct`